src/main/java/org/alliancegenome/curation_api/model/entities/VocabularyTermSet.java

```diff
@@ -50,6 +52,12 @@ public class VocabularyTermSet extends GeneratedAuditedObject {
 	@KeywordField(name = "name_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
 	@JsonView({ View.FieldsOnly.class })
 	private String name;
+	
+	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
+	@KeywordField(name = "vocabularyLabel_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
+	@Column(unique = true)
+	@JsonView({ View.FieldsOnly.class })
+	private String vocabularyLabel;
```

Comment:
We could, but I'm inclined to keep it the same for both Vocabulary and VocabularyTermSet - avoids duplicated code for a field that has the same function

Reply:
I'd planned to just use `label` but that's already used in the Image class in LinkML and since I wanted the field to be required for Vocabulary and VocabularyTermSet it seemed clearer to define a new slot rather than using the slot_usage properties to specify required on each class.

Reply:
What is the usage for "label" is it a string field? Maybe we could just use it?

Reply:
We could, but then we would have to define slot_usage properties on the Vocabulary and VocabularyTermSet classes in order to make them required.  Also, I wanted to keep this slot separate so that the slot description in the LinkML was clear as to what it's purpose was as opposed to the `name` slot.  If `vocabularyLabel` proves to be confusing then we can change it later, but I'd rather keep it as is for now in order to avoid another LinkML release and more code changes.